plugins/importexport/csv/CSVImportExportPlugin.inc.php

```diff
@@ -16,6 +16,117 @@
 import('lib.pkp.classes.plugins.ImportExportPlugin');
 
 class CSVImportExportPlugin extends ImportExportPlugin {
+	/**
+	 * The expected headers coming from the CSV file, in their respective order
+	 *
+	 * @var string[]
+	 */
+	private $_expectedHeaders = [
+		'pressPath',
+		'authorString',
+		'title',
+		'abstract',
+		'seriesPath',
+		'year',
+		'isEditedVolume',
+		'locale',
+		'filename',
+		'doi',
+		'keywords',
+		'subjects',
+		'bookCoverImage',
+		'bookCoverImageAltText',
+		'categories',
+	];
+
+	/**
+	 * The expected size for a valid Submission row on CSV file
+	 *
+	 * @var int
+	 */
+	private $_expectedRowSize;
+
+	/**
+	 * Array for caching already initialized DAOs.
+	 *
+	 * @var DAO[]
+	 */
+	private $_daos = [];
+
+	/**
+	 * Array for caching already retreived Presses.
```

Comment:
There are more occurrences of this typo

```suggestion
	 * Array for caching already retrieved Presses.
```